File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWith2DBarcodes/Utility2DBarcodeFeatures/HideCodeText.php
<?php
namespace Aspose\Barcode\WorkingWith2DBarcodes\Utility2DBarcodeFeatures;

use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\Symbology as Symbology;
use com\aspose\barcode\CodeLocation as CodeLocation;

use java\awt\Font as Font;
class HideCodeText{

    public static function run($dataDir = null)
    {

        # Instantiate barcode object
        $builder = new BarCodeBuilder();

        $symbology=new Symbology();
        $builder->setSymbologyType($symbology->DataMatrix);

        $builder->setCodeText("The quick brown fox jumps over the lazy dog\n The quick brown fox jumps over the lazy dog\n");

        $codeLocation=new CodeLocation();
        $builder->setCodeLocation($codeLocation->None);

        $font = new Font();

        $builder->setCodeTextFont(new Font("Serif", $font->BOLD . $font->ITALIC, 20));

        # Save the image
        $builder->save($dataDir . "HideCodeText.jpg");

        # Display Status
        print "Hide Code Text Successfully.".PHP_EOL;
    }
}

File: Plugins/Aspose.BarCode Java for Jython/asposebarcode/WorkingWith2DBarcodes/Utility2DBarcodeFeatures/HideCodeText.py
from asposebarcode import Settings
from com.aspose.barcode import BarCodeBuilder
from com.aspose.barcode import Symbology
from com.aspose.barcode import CodeLocation
from java.awt import Font

class HideCodeText:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWith2DBarcodes/Utility2DBarcodeFeatures/HideCodeText/'
        
        # Instantiate barcode object
        builder =  BarCodeBuilder()

        symbology= Symbology
        builder.setSymbologyType(symbology.DataMatrix)

        builder.setCodeText("The quick brown fox jumps over the lazy dog\n The quick brown fox jumps over the lazy dog\n")

        codeLocation= CodeLocation
        builder.setCodeLocation(codeLocation.None)

        font =  Font

        builder.setCodeTextFont( Font("Serif", font.BOLD + font.ITALIC, 20))

        # Save the image
        builder.save(dataDir + "HideCodeText.jpg")

        # Display Status
        print "Hide Code Text Successfully."


if __name__ == '__main__':        
    HideCodeText()